src/ulid.cr

```diff
@@ -19,6 +20,53 @@ module ULID
     encode_time(seed_time, TIME_LEN) + encode_random(RANDOM_LEN)
   end
 
+  # Validate a string is a ULID
+  #
+  # ```
+  # ULID.valid?("01B3EAF48P97R8MP9WS6MHDTZ3")
+  # # => true
+  # ```
+  def valid?(ulid : String) : Bool
+    # Incorrect length
+    return false unless ulid.size == TIME_LEN + RANDOM_LEN
+    # Invalid chars
+    return false unless ulid.upcase.chars.all? &.in?(ENCODING)
+
+    return true
```

Comment:
I would do this if we are keeping the same structure

```suggestion
    # Incorrect length || Invalid chars
    return false unless ulid.size == TIME_LEN + RANDOM_LEN || ulid.upcase.chars.all? &.in?(ENCODING)
    
    true
```

Reply:
or even...
```suggestion
    ulid.size == TIME_LEN + RANDOM_LEN && ulid.upcase.chars.all?
```

Reply:
hahaha I think I dropped my brain somewhere today